docs-ref-conceptual/azure-cli-extensions-overview.md

```diff
@@ -43,6 +45,22 @@ az extension add --source <URL-or-path>
 Once an extension is installed, it's found under the value of the `$AZURE_EXTENSION_DIR` shell variable. If this variable is unset, by default the value is `$HOME/.azure/cliextensions` on
 Linux and macOS, and `%USERPROFILE%\.azure\cliextensions` on Windows.
 
+### Install extensions automatically
+
+Azure CLI also has the ability to install an extension dynamically based on the command you run if the command is from an extension that is not installed yet. You can enable the feature through configuration:
```

Comment:
When you run an extension command that is not installed, the Azure CLI can recognize the command you run, and automatically install the extension for you.  This feature, referred to as **dynamic install**, is enabled through configuration.